scripts/mkcloud

```diff
@@ -1095,6 +1095,21 @@ function crowbarupgrade_5plus()
     fi
 }
 
+function cloud6upgrade()
```

Comment:
So it is possible to execute both kind of upgrade jobs on cloud6, the original and the new (unfinished). 

Although there's ongoing discussion (in https://github.com/crowbar/crowbar-core/pull/662) if it actually makes sense.